fs/info.py

```diff
@@ -71,6 +69,15 @@ def get(self, namespace, key, default=None):
         except KeyError:
             return default
 
+    def _require_namespace(self, namespace):
```

Comment:
> Although I'm not sure to understand: why 3 return values ?

The `islink` method could return `None` to indicate the namespace was missing (in addition to False and True). The alternative would be to return `False` if the namespace didn't exist. I wasn't happy with either...